c10/core/TensorOptions.h

```diff
@@ -228,14 +224,6 @@ struct C10_API TensorOptions {
     return r;
   }
 
-  /// Sets the `is_variable` property on the `TensorOptions`.
-  C10_NODISCARD TensorOptions is_variable(c10::optional<bool> is_variable) const noexcept {
-    TensorOptions r = *this;
-    r.set_is_variable(is_variable);
-    return r;
-  }
```

Comment:
We probably need to declare BC-breaking on the removal of the `is_variable` accessors because they are public API (which should be a minor issue because I don't think libtorch end users are actually using it).